Makefile.am

```diff
@@ -228,7 +254,7 @@ collectd_SOURCES = \
 
 
 collectd_CFLAGS = $(AM_CFLAGS)
-collectd_CPPFLAGS = $(AM_CPPFLAGS)
+collectd_CPPFLAGS = $(AM_CPPFLAGS) $(LTDLINCL)
```

Comment:
what does this variable do? if it's for libltdl, we don't use that anymore.

Reply:
Yup, I copied this over from an old commit and didn't realize it wasn't needed anymore. Removed.